Sources/DocCDocumentation/DocCDocumentation.docc/RevenueCat.md

```diff
@@ -55,7 +55,7 @@ Or browse our iOS sample apps:
 
 ### Configuring the SDK
 - ``Purchases/configure(withAPIKey:)``
-- ``Purchases/configure(with:)``
+- ``Purchases/configure(with:)-6oipy``
```

Comment:
This helps DocC resolve the overload.
I decided not to add the new one here since it doesn't really add anything new, it's just a shortcut.

Reply:
So it doesn't show up like that in the docs? Cool.

Reply:
No hehe, it'll just show the right overload type:
![Uploading Screen Shot 2022-06-21 at 12.33.28.png…]()

Reply:
That's what I thought, just had to double check cause I'm not familiar with how it works 😄